datahub-web-react/src/app/ingest/source/builder/RecipeForm/ConnectionRequest.tsx

```diff
@@ -0,0 +1,266 @@
+import { CheckCircleOutlined, CloseCircleOutlined } from '@ant-design/icons';
+import { Button, message, Modal, Typography } from 'antd';
+import React, { useEffect, useState } from 'react';
+import { green, red } from '@ant-design/colors';
+import styled from 'styled-components/macro';
+import { ReactComponent as LoadingSvg } from '../../../../../images/datahub-logo-color-loading_pendulum.svg';
+import { ANTD_GRAY } from '../../../../entity/shared/constants';
+import {
+    useCreateTestConnectionRequestMutation,
+    useGetIngestionExecutionRequestLazyQuery,
+} from '../../../../../graphql/ingestion.generated';
+import { RUNNING, yamlToJson } from '../../utils';
+import ConnectionCapabilityView from './ConnectionCapabilityView';
+
+const LoadingWrapper = styled.div`
+    display: flex;
+    justify-content: center;
+    margin: 50px 0 60px 0;
+`;
+
+const LoadingSubheader = styled.div`
+    display: flex;
+    justify-content: center;
+    font-size: 12px;
+`;
+
+const LoadingHeader = styled(Typography.Title)`
+    display: flex;
+    justify-content: center;
+`;
+
+const ResultsHeader = styled.div`
+    align-items: center;
+    display: flex;
+    justify-content: center;
+    margin-bottom: 24px;
+    font-size: 22px;
+
+    svg {
+        margin-right: 6px;
+    }
+`;
+
+const StatusWrapper = styled.span<{ color: string }>`
+    color: ${(props) => props.color};
+    font-weight: bold;
+`;
+
+const ResultsWrapper = styled.div`
+    background-color: ${ANTD_GRAY[3]};
+    border-radius: 4px;
+    padding: 16px 24px;
+`;
+
+enum SourceCapability {
+    PLATFORM_INSTANCE = 'Platform Instance',
+    DOMAINS = 'Domains',
+    DATA_PROFILING = 'Data Profiling',
+    USAGE_STATS = 'Dataset Usage',
```

Comment:
What if we just called this "Usage Stats"

Reply:
sounds good